application/controllers/collection_instrument.py

```diff
@@ -85,20 +86,30 @@ def upload_instrument(self, exercise_id, file, ru_ref=None, classifiers=None, se
         :return: a collection instrument instance
         """
 
+        if current_app.config["SEFT_CI_DATABASE_TABLE_DEPRECATED"] is True:
+            try:
+                seft_ci_bucket = GoogleCloudSEFTCIBucket(current_app.config)
+                path = seft_ci_bucket.upload_file_to_bucket(file=file)
+            except Exception:
+                log.exception("An error occurred when trying to put SEFT CI in bucket")
+
         log.info("Upload exercise", exercise_id=exercise_id)
 
         validate_uuid(exercise_id)
         instrument = InstrumentModel(ci_type="SEFT")
 
```

Comment:
I think the next two lines need to stay if we're allowing the feature to be toggled on or off, otherwise the old database stuff won't work

Reply:
and wrapped in the feature flag accordingly